src/Microsoft.AspNetCore.Sockets.Common/Formatters/MessageFormatter.cs

```diff
@@ -7,14 +7,20 @@ namespace Microsoft.AspNetCore.Sockets.Formatters
 {
     public static class MessageFormatter
     {
+        public static readonly byte TextFormatIndicator = (byte)'T';
+        public static readonly byte BinaryFormatIndicator = (byte)'B';
+
+        public static readonly string TextContentType = "application/vnd.microsoft.aspnetcore.endpoint-messages.v1+text";
+        public static readonly string BinaryContentType = "application/vnd.microsoft.aspnetcore.endpoint-messages.v1+binary";
+
         public static bool TryFormatMessage(Message message, Span<byte> buffer, MessageFormat format, out int bytesWritten)
         {
             if (!message.EndOfMessage)
             {
-                // This is a truely exceptional condition since we EXPECT callers to have already
+                // This is truly an exceptional condition since we EXPECT callers to have already
                 // buffered incomplete messages and synthesized the correct, complete message before
                 // giving it to us. Hence we throw, instead of returning false.
-                throw new InvalidOperationException("Cannot format message where endOfMessage is false using this format");
+                throw new ArgumentException("Cannot format message where endOfMessage is false using this format", nameof(message));
```

Comment:
I changed it because the exception is due to an invalid argument, so it felt right.